README.rst

```diff
@@ -1,16 +1,16 @@
-celer
+Celer
```

Comment:
can you keep `celer` ? (consistency with the package name matters more to me than consistency with the paper)